src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadoctype/InputMissingJavadocTypeInner.java

```diff
@@ -0,0 +1,79 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2001
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadoctype;
+
+/**
+ * Tests having inner types
+ * @author Oliver Burn
```

Comment:
please remove this line,

Can we make inputs in our new Input format ?  (config is in javadoc of class, "warn"  as trailing comments)

Reply:
Original check verified author which is why this is in the examples. I removed them since this check doesn't mind them/

> Can we make inputs in our new Input format ?

I will do the best I can but we use the same input with different configs sometimes, so it is not always cut and dry warn.

Reply:
Only 3 files could changed. The rest use the file more then once and have different violations.
Let me know if there is anything else you want.